### DISCUSSION TITLE
How to setup data importer

### DISCUSSION THREAD

**Discussion by QingbiaoLi** [firefly-iii/firefly-iii#7023] [Firefly III]
Hello, 

I am new to firefly-iii. Thank you for creating this great tool.

I am trying to set up fireflyiii via docker on Synology NAS.
https://tzamtzis.gr/2020/how-to/firefly-synology-docker/

However, we need to use https for data importer but I only able to set up https
https://192.168.0.52:8080
It works in the main page but it cannot direct into https://192.168.0.52:8081/preferences
http://192.168.0.52:8081/preferences

It works if it host at http://192.168.0.52:8080

**Comment by QingbiaoLi** [comment]
{"error":"unsupported_grant_type","error_description":"The authorization grant type is not supported by the authorization server.","hint":"Check that all required parameters have been provided","message":"The authorization grant type is not supported by the authorization server."}

**Reply by JC5** [reply]
I'd love to help but I'm not sure what the question is.
